### ISSUE TITLE
Display Variable Frequencies and Percentages in Viewing Mode

### ISSUE THREAD

**Issue by sandaionescu** [ncrncornell/ced2ar#12]
variable frequencies only display in edit mode. They should display in the "public" view as well.

**Comment by CSimmer** [comment]
Can you please provide links/screenshots where you see and where you don't.
Thanks in advance,

**Comment by sandaionescu** [comment]
I *do not* see category level frequencies here: 
https://demo.ncrn.cornell.edu/ced2ar-web/codebooks/icpsr07213/v/4/vars/V520013
But I *do* see them here:
https://demo.ncrn.cornell.edu/ced2ar-web/edit/codebooks/icpsr07213/v/4/vars/V520013

**Comment by CSimmer** [comment]
View/public page with NO category level frequencies:
https://demo.ncrn.cornell.edu/ced2ar-web/codebooks/icpsr07213/v/4/vars/V520013

![ss codebook - no stats 2017-03-14](https://cloud.githubusercontent.com/assets/22204663/23917833/1658624e-08c7-11e7-8a85-7e3337e1bc3e.png)

**Comment by CSimmer** [comment]
Edit page with category level frequencies:
https://demo.ncrn.cornell.edu/ced2ar-web/edit/codebooks/icpsr07213/v/4/vars/V520013

![ss edit codebook - stats 2017-03-14](https://cloud.githubusercontent.com/assets/22204663/23917899/4abfd9b8-08c7-11e7-806e-bcedee57c034.png)

**Comment by CSimmer** [comment]
You only mentioned frequencies in the issue. Do you want the percentags too?

**Comment by sandaionescu** [comment]
Yes, sorry. I do want percentages as well. Since they are in the DDI, and are displayed in edit mode, they should be displayed in "view" mode as well.

**Comment by CSimmer** [comment]
Updated Issue title.  Added " and Percentages" to make sure it is included.

**Comment by CSimmer** [comment]
General rule per Lars: 
We should not have discrepancies between edit and view screens. That's ours to fix.
